aten/src/ATen/core/jit_type.h

```diff
@@ -1600,6 +1600,39 @@ CAFFE2_API TypePtr tryEvalTypeVariables(TypePtr type, TypeEnv& type_env);
 
 CAFFE2_API bool elementTypeCanBeInferredFromMembers(const TypePtr& elem_type);
 
+enum ATTRIBUTE_KIND_ENUM {
```

Comment:
nit: while we are pretty loose with style, theoretically we follow fb style. So this should be `AttributeKindEnum`. 

Also: should use `enum class`